pkg/loader/compose/compose.go

```diff
@@ -178,6 +183,50 @@ func loadEnvVars(envars []string) []kobject.EnvVar {
 	return envs
 }
 
+// Convert the Docker Compose v3 volumes to []string (the old way)
+// TODO: Check to see if it's a "bind" or "volume". Ignore for now.
+// See: https://docs.docker.com/compose/compose-file/#long-syntax-2
+func loadV3Volumes(volumes []types.ServiceVolumeConfig) []string {
```

Comment:
rather than degrading the information that we are getting from docker-compose, can we not improve our data structures?